src/components/Scrollbar.tsx

```diff
@@ -32,15 +33,16 @@ export default function ({
   children,
   fade = 'both',
   parentHeight = 0,
-  extraPadding = false,
+  bottomPadding = 0,
+  titlePadding = 0,
 }: ChildrenProp & ScrollbarProps) {
   const { height = 0, ref } = useResizeDetector({ handleWidth: false })
 
   const thumbRef = useRef() as MutableRef<HTMLDivElement>
   const [thumbHeight, setThumbHeight] = useState(100)
-  const extaHeightParent = parentHeight > 370 ? 40 : 0
-  const extaHeightTitle = extraPadding ? 32 : 0
-  const extraReservedSpace = extaHeightParent + extaHeightTitle
+  const extaBottomPadding = parentHeight > 370 ? bottomPadding : 0
```

Comment:
What is `370`? Looks like a magic number

Reply:
It's a maximum height that the scrollbar block may have. Because of static card height, we have to limit the maximum height by design